azurerm/resource_arm_automation_schedule.go

```diff
@@ -107,14 +108,94 @@ func resourceArmAutomationSchedule() *schema.Resource {
 				//todo figure out how to validate this properly
 			},
 
-			//todo missing properties: week_days, month_days, month_week_day from advanced automation section
+			"advanced_schedule": {
```

Comment:
>"advanced_schedule" [](start = 3, length = 19)

*suggestion* We don't need to introduce wrapper `advanced_schedule` object here. What I mean is to move the mutual exclusive objects `week_days`, `month_days` and `monthly_occurrence` to the top level so we can reduce the depth of the hierarchy.